socketio/src/packet.rs

```diff
@@ -96,50 +97,47 @@ impl From<&Packet> for Bytes {
     /// stream as it gets handled and send by it's own logic via the socket.
     fn from(packet: &Packet) -> Bytes {
         // first the packet type
-        let mut string = (packet.packet_type as u8).to_string();
+        let mut buffer = String::with_capacity(64);
```

Comment:
Can you please add a comment on why you settled for 64 here?

Reply:
Good point, I forgot that there's not necessarily attachments and allocated for them either way. Probably wiser to not pre-allocate all.